src/components/fair/booth.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+
+import Artworks from "../artwork_grid"
+import Button from "../buttons/ghost"
+import Text from "../text"
+
+const PageSize = 10
+
+interface Props extends RelayProps, React.HTMLProps<FairBooth> {
+  show: any
+  onContactGallery?: () => any,
```

Comment:
Should call this callback with the partner ID / whatever else is needed by Force’s modal.